app/controllers/home.js

```diff
@@ -51,14 +59,20 @@ export default class HomeController extends Controller {
     this.theme.setTheme(theme);
   }
 
+  @action
+  setWeather(event) {
+    this.weather.currentWeather = event.target.value;
+    this.eventBus.publish("weatherChanged", event.target.value);
```

Comment:
not sure why `setWeather` is in this file

Reply:
oh i see what's happening nvm

Reply:
all good. maybe it would be smarter to have a `setWeather()` method like the theme service and then update the event bus with that instead of duplicating this code in site-settings